config.cordovabuild.xml

```diff
@@ -36,6 +36,8 @@
         <param name="ios-package" onload="true" value="CDVStatusBar" />
     </feature>
     <platform name="ios">
+        <preference name="deployment-target" value="12.0"/>
```

Comment:
we actually want the deployment target to be 13. That is the minimum version that we have supported for a while to get our javascript to work.